.env.test

```diff
@@ -0,0 +1 @@
+CI=true
```

Comment:
how is this run? in what environment?

Reply:
It depends on the command. Build uses `production`, etc.

Reply:
Removed file.